FreeRTOS/Demo/CORTEX_MPU_M3_MPS2_QEMU_GCC/init/startup.c

```diff
@@ -122,7 +122,8 @@ void Default_Handler( void )
         "NVIC_INT_CTRL_CONST: .word 0xe000ed04\n"
     );
 }
-static void HardFault_Handler( void ) __attribute__( ( naked ) );
+
+static __attribute__( ( naked ) )
```

Comment:
Same as above.

Reply:
Thanks a lot for all your comments. I'll update the patch with a new one here for further review.

best regards,

Florian La Roche